.github/workflows/wp-product-info.yml

```diff
@@ -0,0 +1,15 @@
+name: Deploy to wp-product-info
+
+on:
+  release:
+    types: [ published ]
```

Comment:
If I remember correctly, a workflow can not call another workflow without some additional config.

https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Since changesets (another workflow) creates our release, I'm not sure this workflow will get triggered. We experienced this before in the faustjs repo.

Reply:
[Here is the PR that we dealt with this on Faust.](https://github.com/wpengine/faustjs/pull/1390). Thankfully it looks like now there is a way to invoke a workflow from a workflow:

https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow